### ISSUE TITLE
[Bug]: Embedded `Table` with `PropertyFilter` styles issue

### ISSUE THREAD

**Issue by nichita-pasecinic** [cloudscape-design/components#503]
### Package version

^3.0.126

### React version

^17.0.2

### Description

Hi,

 An `Table` component with prop `variant={'embedded'}` and filter component like so:
```jsx
filter={
  <PropertyFilter
    i18nStrings={PROPERTY_FILTERING_I18N_CONSTANTS}
    countText={filterCountText}
    {...propertyFilterProps}
    expandToViewport
  />
}
```

has some weird styles:

![image](https://user-images.githubusercontent.com/98735377/202920129-5b841d0a-265a-4160-a13f-643f8c6f91c3.png)

Is this expected ?

### Source code

```jsx
<Table
      {...collectionProps}
      variant={'embedded'}
      items={items}
      columnDefinitions={columnDefinitions}
      resizableColumns
      filter={
        <PropertyFilter
          i18nStrings={PROPERTY_FILTERING_I18N_CONSTANTS}
          countText={filterCountText}
          {...propertyFilterProps}
          expandToViewport
        />
      }
    />
```

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/cloudscape-design/components/blob/main/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/cloudscape-design/components/issues) for duplicate problems

**Comment by connorlanigan** [comment]
Hello,

the Table in your screenshot looks similar to [the "Embedded table" playground example on our website](https://cloudscape.design/components/table/?tabId=playground&example=embedded-table). Can you describe how the table in your screenshot differs from what you expect?

**Comment by nichita-pasecinic** [comment]
Hi @connorlanigan 
this part (border radiuses/corners) looks odd to me 
![image](https://user-images.githubusercontent.com/98735377/203054850-24051a7d-f075-4ca7-a159-aabd79d1edac.png)

**Comment by connorlanigan** [comment]
Ah, I see, thank you. I will bring this up with the team.

**Comment by timogasda** [comment]
Hello @nichita-pasecinic !

Sorry for the late response. This should have been fixed with #529. Please update your Cloudscape version and try again.
